openhtf/core/test_executor.py

```diff
@@ -96,9 +101,11 @@ def __init__(self, test_descriptor: 'test_descriptor.TestDescriptor',
                execution_uid: Text,
                test_start: Optional[phase_descriptor.PhaseDescriptor],
                test_options: 'test_descriptor.TestOptions',
-               run_phases_with_profiling: bool):
+               run_phases_with_profiling: bool, 
+               plug_manager: Optional[PlugManager] = None):
     super(TestExecutor, self).__init__(name='TestExecutorThread')
```

Comment:
Would there be any value in adding the "slot id" in so that it can be in the thread name? Maybe only useful for debug?